src/utils/mavenUtils.ts

```diff
@@ -9,13 +9,22 @@ import { TelemetryProperties } from "vscode-azureextensionui";
 import * as xml2js from 'xml2js';
 import { localize } from '../localize';
 import { cpUtils } from './cpUtils';
+import { ext } from '../extensionVariables';
+import { DialogResponses } from 'vscode-azureextensionui';
+
+import opn = require("opn");
 
 export namespace mavenUtils {
     const mvnCommand: string = 'mvn';
     export async function validateMavenInstalled(workingDirectory: string | undefined): Promise<void> {
         try {
             await cpUtils.executeCommand(undefined, workingDirectory, mvnCommand, '--version');
         } catch (error) {
+            const message: string = localize('azFunc.mvnNotFound', 'Failed to find "maven", please set the maven path');
+            const result: vscode.MessageItem = await ext.ui.showWarningMessage(message, DialogResponses.learnMore, DialogResponses.dontWarnAgain);
```

Comment:
We need to make some action when the user selects `DialogResponses.dontWarnAgain`

Reply:
I plan to throw an Error no matter user selects learnMore or dontWarnAgain, in order to interrupt the execution of function which invoke validateMavenInstalled, so i didn't add another if but just throw an Error behind it.